js/DataService.js

```diff
@@ -8,13 +8,46 @@ angular.module('PathOfDamage')
     var int = parseInt(string);
     return isNaN(int) ? null : int;
   }
+  
+  function Row(key, value) {
+    this.enabled = true;
+    this.name = '';
+    this.value = null;
+    if (key !== undefined) {
```

Comment:
What bothers you about it? I like having a centralized idea of a row with defaults like this.